src/vendor/ledger-bitcoin/src/buffertools.ts

```diff
@@ -0,0 +1,134 @@
+import { createVarint, parseVarint, sanitizeBigintToNumber } from "./varint";
+
+export function unsafeTo64bitLE(n: number): Buffer {
+  // we want to represent the input as a 8-bytes array
+  if (n > Number.MAX_SAFE_INTEGER) {
```

Comment:
check if n < 0?

Also, Idk enough about typescript to know, but will we ever need to represent numbers > Number.MAX_SAFE_INTEGER ?

Reply:
tbh, they might take PRs if you wanted to suggest, but also i imagine you might enjoy taking a look at their rust code too if you were curious.